register.js

```diff
@@ -1,4 +1,4 @@
 const { register } = require('node:module')
 const { pathToFileURL } = require('node:url')
 
-register('./loader-hook.mjs', pathToFileURL(__filename))
+register('./loader-hook.mjs', pathToFileURL(__filename), { data: { include: ['openai'] } })
```

Comment:
it looks like this breaks other things, ie `vitest` integration tests are failing ... i'm not 100% on this approach, or if there's a different way I should be specifying this